superset-frontend/src/views/datasetList/DatasetList.tsx

```diff
@@ -277,31 +275,16 @@ const DatasetList: FunctionComponent<DatasetListProps> = ({
     {
       Header: t('Schema'),
       accessor: 'schema',
-      disableSortBy: true,
       size: 'lg',
     },
     {
       Cell: ({
         row: {
-          original: { changed_on: changedOn },
+          original: { changed_on_delta_humanized: changedOn },
         },
-      }: any) => {
-        const momentTime = moment(changedOn);
-        const time = momentTime.format(SHORT_DATE);
-        const date = momentTime.format(SHORT_TIME);
-        return (
-          <TooltipWrapper
-            label="last-modified"
-            tooltip={time}
-            placement="right"
-          >
-            <span>{date}</span>
-          </TooltipWrapper>
-        );
-      },
-      Header: t('Last Modified'),
-      accessor: 'changed_on',
-      size: 'xl',
+      }: any) => <span className="no-wrap">{changedOn}</span>,
+      Header: t('Modified'),
+      accessor: 'changed_on_delta_humanized',
```

Comment:
can you add back the `size: 'xl'` property which should ensure even spacing between columns